plugins/com.google.cloud.tools.eclipse.appengine.validation/plugin.xml

```diff
@@ -118,7 +118,7 @@
 		  scope="total"
 		  class="com.google.cloud.tools.eclipse.appengine.validation.AppEngineWebXmlSourceValidator"
 		  id="appEngineWebXmlSourceValidator">
-		<contentTypeIdentifier id="appengine-web.xml">
+		<contentTypeIdentifier id="com.google.appengine.dd.xml">
```

Comment:
What's dd? deployment descriptor? That usually refers to web.xml, not appengine-web.xml:

https://cloud.google.com/appengine/docs/standard/java/config/webxml

If this is validating web.xml then the class/ID should probably be  WebXmlSourceValidator rather than AppEngineWebXmlSourceValidator